renku_notebooks/api/notebooks.py

```diff
@@ -41,6 +42,36 @@
 bp = Blueprint("notebooks_blueprint", __name__, url_prefix=config.SERVICE_PREFIX)
 
 
+@bp.route("/version")
+def version():
+    """
+    Return notebook services version.
+
+    ---
+    get:
+      description: Information about notebooks service.
+      responses:
+        200:
+          description: Notebooks service info.
+          content:
+            application/json:
+              schema: VersionResponse
+    """
+    info = {
+        "name": "renku-notebooks",
+        "versions": [
+            {
+                "version": config.NOTEBOOKS_SERVICE_VERSION,
+                "data": {
+                    "anonymousSessionsEnabled": config.ANONYMOUS_SESSIONS_ENABLED,
+                    "cloudstorageEnabled": config.S3_MOUNTS_ENABLED,
```

Comment:
@mohammad-sdsc this is supposed to have one more level of keys that indicate the type of storage.

```suggestion
                    "cloudstorageEnabled": {
                        "s3": config.S3_MOUNTS_ENABLED,
                    },
```